extensions/examples/online-extensions.js

```diff
@@ -41,8 +41,8 @@ angular
     'extensionRegistry',
     function(extensionRegistry) {
 
-      var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn">');
-      var system_status_elem_mobile = $('<div row flex class="navbar-flex-btn system-status-mobile">');
+      var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn" style="display: none;">');
+      var system_status_elem_mobile = $('<div row flex class="navbar-flex-btn system-status-mobile" style="display: none;">');
```

Comment:
Agreed.  I'm unclear, too, but that is what is [in production for online](https://github.com/openshift/online/blob/master/ui/assets/extensions/online-extensions.js#L102-L118), so I figured it was better to match that since that's the real world use case?  It looks like this example has been more recently updated than what's in online, so maybe that's a bad assumption?  @benjaminapetersen, it looks like [you removed the inline display: none styles and javascript style undo](https://github.com/rhamilto/origin-web-console/blame/7f8960af97f07793c657f9f64b1b3d6fc1c5e988/extensions/examples/online-extensions.js#L55-L59).  Thoughts?